RELEASE_NOTES.md

```diff
@@ -14,6 +14,8 @@
 
 ### New hardware support
 
+* CSM 1.6 does not support servers with NVIDIA CPUs and GPUs. Systems with these servers should not upgrade to CSM 1.6.
```

Comment:
Wondering if we should we use "1.6.0" instead of "1.6" since it sounds like the "1.6.1" release will have support.  Some folks might interpret "1.6" as "1.6.*"?